docs/users-guide/04-asking-questions.md

```diff
@@ -24,7 +24,40 @@ To ask a question about a table of data, we usually do some **filtering** and/or
 
 Filtering just means narrowing things down based on certain criteria. You're probably already familiar with filtering when looking for something online, like when shopping. Maybe you only want to see olive-colored pants, or books where the author's last name is "Steinbeck," or pictures of people wearing olive-colored pants reading John Steinbeck.
 
-You can do the same kind of thing with data in Metabase. Just click the Filter button in the top-right of the screen to open the filter sidebar. You'll see a list of all of the columns in this table, as well as columns from tables that are related to the one you're looking at. Depending on the column you pick, you'll see slightly different options for your filter.
+### Using the Question Builder interface
```

Comment:
This merge conflict wasn't resolved correctly. This is old copy that includes images of the old QB bar.